features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Environments/Environments.jsx

```diff
@@ -274,6 +275,9 @@ const useStyles = makeStyles((theme) => ({
     vhostSelect: {
         marginTop: theme.spacing(3),
     },
+    textCount: {
+        marginTop: '-20px',
```

Comment:
Should use theme spacing or a `<Box mt={-3}>`

Reply:
Fixed